### PR TITLE
Add new SINGULARITY_ env vars, from sylabs 59

### PR DESCRIPTION
**Pull request by edytuk**
It takes a little bit from sylabs PR

- sylabs/singularity-userdocs#59

### CODE REVIEW

File: appendix.rst [hunk 1]
```diff
@@ -70,7 +70,17 @@ You can see them listed alphabetically below with their respective functionality
 
 #. **SINGULARITY_DOCKER_PASSWORD**: To specify the password for docker authentication.
 
-#. **SINGULARITY_DROP_CAPS**: To specify a list (comma separated string) of capabilities to be dropped. Default is an empty string.
+#. **SINGULARITY_DOWNLOAD_CONCURRENCY**: To specify how many concurrent streams
```
Comment (DrDaveD) [inline appendix.rst:73]: These 3 will need to be removed when the library interface is removed, but for now they match what is in the apptainer master branch.

File: appendix.rst [hunk 2]
```diff
@@ -147,7 +164,11 @@ You can see them listed alphabetically below with their respective functionality
 
 #. **SINGULARITY_NO_PRIVS**: To drop all the privileges from root user in the container. Default is set to false.
 
-#. **SINGULARITY_NV**: To enable experimental Nvidia support. Default is set to false.
+#. **SINGULARITY_NV**: To enable Nvidia GPU support. Default is
+   set to false.
+
+#. **SINGULARITY_NVCCLI**: To use nvidia-container-cli for container GPU setup
+   (experimental).
```
Comment (cclerget) [inline appendix.rst]: For apptainer, this option is only supported for unprivileged run (user namespace) and root user, a note mentioning that would avoid some confusion

Comment (DrDaveD) [reply appendix.rst]:
```suggestion
   (experimental, only unprivileged).
```

**Comment by DrDaveD** [comment]
Since this is 4 commits, when we merge we should use "squash and merge"

**Review by cclerget — APPROVED** [review]

**Pull request merged**